core/domain/question_services.py

```diff
@@ -85,31 +86,113 @@ def _create_new_question(committer_id, question, commit_message):
     create_question_summary(question.id, committer_id)
 
 
-def create_new_question_skill_link(question_id, skill_id, skill_difficulty):
-    """Creates a new QuestionSkillLink model.
+def create_multi_question_skill_links_for_question(
+        user_id, question_id, skill_ids, skill_difficulties):
+    """Creates multiple new QuestionSkillLink models. Also adds
```

Comment:
This is talking about two separate disparate actions -- can we preface this with a more unified description about what's going on here conceptually? I.e. is it adding a new question, or adding new skills to a question, or something like that?

Also, maybe the function name should change accordingly. (As an external caller I don't really understand _when_ I should be calling this function.)

Reply:
Done